src/_guides/language/effective-dart/design.md

```diff
@@ -260,18 +260,56 @@ property, and should be named as such using a phrase that describes *what* the
 member returns.
 
 {:.good-style}
-<?code-excerpt "misc/lib/effective_dart/design_good.dart (noun-for-func-returning-value)" replace="/;//g"?>
+<?code-excerpt "misc/lib/effective_dart/design_good.dart (noun-for-func-returning-value)"?>
 {% prettify dart %}
-list.elementAt(3)
-list.firstWhere(test)
-string.codeUnitAt(4)
+var element = list.elementAt(3);
+var first = list.firstWhere(test);
+var char = string.codeUnitAt(4);
 {% endprettify %}
 
 This guideline is deliberately softer than the previous one. Sometimes a method
 has no side effects but is still simpler to name with a verb phrase like
 `list.take()` or `string.split()`.
 
 
+### CONSIDER an imperative verb phrase for a function or method if you want to draw attention to the work it performs.
+
+When a member produces a result without any side effects, it should usually be a
+getter or a method with noun phrase name describing the result it returns.
+However, sometimes the work required to produce that result is important. It may
+be prone to runtime failures, or use heavyweight resources like networking or
+file IO. In cases like this, where you want the caller to think about the work
+the member is doing, give the member a verb phrase name that describes that
+work.
+
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/design_good.dart (verb-for-func-with-work)"?>
+{% prettify dart %}
+var table = database.downloadData();
+var packageVersions = packageGraph.solveConstraints();
+{% endprettify %}
+
+Note, though, that this guideline is softer than the previous two. The work an
+operation performs is often an implementation detail that isn't relevant to the
+caller, and performance and robustness boundaries change over time. Most of the
+time, name your members based on *what* it does for the caller, not *how* it
+does it.
+
+
+### AVOID starting a method name with `get`.
+
+In most cases, the method should be an actual getter whose name is whatever you
```

Comment:
I found this hard to read. Maybe:

the method ... -> a method that you're tempted to name `getXyz` should be an actual getter with the name `xyz`. (and then drop the example, maybe)

(I see we use ___ else for a stand-in name, but that would be bad at the beginning of a member name.)

We could use italics to indicate that the name is a placeholder.

Reply:
Agree it was awkward. Tried something a little different.